src/plugins/embeddable/public/lib/panel/panel_header/panel_actions/add_panel/add_panel_flyout.tsx

```diff
@@ -163,12 +168,21 @@ export class AddPanelFlyout extends React.Component<Props> {
         <EuiFlyoutFooter>
           <EuiFlexGroup justifyContent="flexEnd">
             <EuiFlexItem grow={true}>
-              <EuiSuperSelect
-                data-test-subj="createNew"
-                options={this.getSelectCreateNewOptions()}
-                valueOfSelected="createNew"
-                onChange={(value: string) => this.createNewEmbeddable(value)}
-              />
+              <EuiPopover
+                id="createNew"
+                button={
+                  <EuiButton data-test-subj="createNew" onClick={this.toggleCreateMenu}>
+                    {createNewLabel}
+                  </EuiButton>
+                }
+                isOpen={this.state.isCreateMenuOpen}
+                closePopover={this.closeCreateMenu}
+                panelPaddingSize="none"
+                anchorPosition="upLeft"
+                withTitle
+              >
+                <EuiContextMenuPanel title={createNewLabel} items={this.getCreateMenuItems()} />
```

Comment:
I agree that "Create new" is duplicated too many times, just remove the title from the context menu since it's still visible from the button text.
```suggestion
                <EuiContextMenuPanel items={this.getCreateMenuItems()} />
```